### PR TITLE
[WIP] Omni remove thing cache in bridge

### PR DESCRIPTION
**Pull request by digitaldan**
Signed-off-by: Dan Cunningham <dan@digitaldan.com>

### CODE REVIEW

**Comment by digitaldan** [comment]
This is in regards to #45 , @boc-tothefuture take a look, I did this quickly and only tested zones.

File: addons/binding/org.openhab.binding.omnilink/src/main/java/org/openhab/binding/omnilink/handler/OmnilinkBridgeHandler.java [hunk 1]
```diff
@@ -266,9 +257,8 @@ public Void call() throws Exception {
     }
 
     private void handleUnitStatus(UnitStatus stat) {
-        Integer number = stat.getNumber();
-        Thing theThing = unitThings.get(number);
-        logger.debug("received status update for unit: " + number + ", status: " + stat.getStatus());
+        Thing theThing = getChildThing(OmnilinkBindingConstants.THING_TYPE_UNIT, stat.getNumber());
+        logger.debug("received status update for unit: " + stat.getNumber() + ", status: " + stat.getStatus());
```
Comment (digitaldan) [inline addons/binding/org.openhab.binding.omnilink/src/main/java/org/openhab/binding/omnilink/handler/OmnilinkBridgeHandler.java]: just an FYI we should replace logging lines like this with something like:
`logger.debug("received status update for unit: {}, status: {} ", stat.getNumber() , stat.getStatus());`

Comment (boc-tothefuture) [inline addons/binding/org.openhab.binding.omnilink/src/main/java/org/openhab/binding/omnilink/handler/OmnilinkBridgeHandler.java]: :+1 
String concat is bad where unnecessary.

**Comment by boc-tothefuture** [comment]
@digitaldan - This looks good to me. What has to be done to remove the WIP so it can be merged?

**Comment by craigham** [comment]
nice fix.  definitely good to get rid of that code.

**Pull request merged**